README.md

```diff
@@ -1174,6 +1184,17 @@ a $A $B=`echo $A`:
   echo $A $B
 ```
 
+When [export](#export) is set, all `just` variables are exported as environment variables.
+
+#### Loading Environment Variables from a `.env` File
+
+`just` will load environment variables from a `.env` file if [dotenv-load](#dotenv-load) is set. The variables in the file will be available as environment variables to the recipes. See [dotenv-integration](#dotenv-integration) for more information.
+
+#### Setting `just` Variables from Environments Variables
+
+Environment variables can be propagated to `just` variables using the functions `env_var()` and `env_var_or_default()`. These variables will be available as environment variables to the recipes.
```

Comment:
I removed the text :
"These variables will be available as environment variables to the recipes."
(that I had added)
to avoid confusion, as environment variables (from the shell) are available to recipes.